pkg/apis/apps/v1alpha1/apimanager_types.go

```diff
@@ -47,7 +47,8 @@ type APIManagerSpec struct {
 	// +optional
 	Zync *ZyncSpec `json:"zync,omitempty"`
 	// +optional
-	HighAvailability *HighAvailabilitySpec `json:"highAvailability,omitempty"`
+	HighAvailability    *HighAvailabilitySpec    `json:"highAvailability,omitempty"`
+	PodDisruptionBudget *PodDisruptionBudgetSpec `json:"podDisruptionBudget,omitempty"`
```

Comment:
I think PodDisruptionBudget should have the // +optional tag comment in this case

Reply:
will fix this.